src/create-ssr-search-client.ts

```diff
@@ -56,7 +56,7 @@ export function createSSRSearchClient({
     ...options,
     requester: {
       send({ headers, method, url, data }) {
-        const transferStateKey = makeStateKey(`ngais(${data})`);
+        const transferStateKey = makeStateKey<string>(`ngais(${data})`);
```

Comment:
do you know what makes string not inferred here?

Reply:
I'm not sure how inference would work here. If we don't tell it what's type used for state, it's fallback to `void` 

```ts
makeStateKey<T = void>(key: string): StateKey<T>
```
docs: https://angular.io/api/platform-browser/makeStateKey

Reply:
ah yes the key isn't the value, makes sense